src/connectors/gds.fm.ts

```diff
@@ -7,3 +7,12 @@ Connector.artistSelector = '.player-meta__artist';
 Connector.trackSelector = '.player-meta__title';
 
 Connector.playButtonSelector = '.player-meta__icon--play';
+
+Connector.scrobblingDisallowedReason = () => {
+	const artist = Connector.getArtist();
+	if (artist && artist.match(/gds\.fm/)) {
+		return 'Announcement';
```

Comment:
Please use a valid DisallowedReason https://github.com/web-scrobbler/web-scrobbler/blob/master/src/core/object/disallowed-reason.ts

In this case 'FilteredTag' seems appropriate at a glance.

Reply:
You're right! Had issues with getting local dev to work, so I didn't find the expected types. Matches what other connectors use in this case.